### ISSUE TITLE
plots diff --help: throws error

### ISSUE THREAD

**Issue by dberenbaum** [iterative/dvc#5459]
# Bug Report
### Reproduce

```console
$dvc plots diff --help
ERROR: unexpected error - unsupported operand type(s) for %: 'tuple' and 'dict'
```

### Environment information

**Output of `dvc version`:**

```console
$ dvc version
DVC version: 2.0.0a0+273b02
---------------------------------
Platform: Python 3.9.1 on macOS-10.16-x86_64-i386-64bit
Supports: http, https
Cache types: <https://error.dvc.org/no-dvc-cache>
Caches: local
Remotes: None
Workspace directory: apfs on /dev/disk3s1s1
Repo: dvc, git
```

**Comment by pmrowla** [comment]
should just need the same change as https://github.com/iterative/dvc/pull/5453 but for plots

**Comment by dberenbaum** [comment]
Fixed by #5492
